models/core/parts/lockpin.scad

```diff
@@ -8,7 +8,7 @@ include <../main.scad>
 /* [Parameters] */
 
 // Type of grip for the lock pin
```

Comment:
since the `extended` lockpin type has some regressions in terms of printability and it will likely (didn't check) block another lockpin from being placed when inside a corner of a connector, we need to mention that here.

Reply:
Agreed